File: Singular.Web.Admin/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using Singular.Configuration;

namespace Singular.Web.Admin.Controllers
{
    /// <summary>
    /// Home controller
    /// </summary>
    public class HomeController : Controller
    {
        /// <summary>
        /// Index - returns dashboard layout
        /// </summary>
        /// <returns></returns>
        public ActionResult Index()
        {
            return View();
        }
    }
}

File: Singular.Web.Admin/Ng/controllers/homeController.js
'use strict';

// HOME CONTROLLER
(function ($a) {

    $a.controller("homeController", ["$scope", function ($scope) {

        $scope.actions = {
            
            index: function(id) {
                console.log("route id is " + id);
            }

        };

    }]);


})(Singular.Application)